internal/http/services/owncloud/ocs/handlers/apps/sharing/shares/public.go

```diff
@@ -423,6 +423,13 @@ func (h *Handler) removePublicShare(w http.ResponseWriter, r *http.Request, shar
 }
 
 func ocPublicPermToCs3(permKey int, h *Handler) (*provider.ResourcePermissions, error) {
+
+	if permKey == 31 {
```

Comment:
Could we use the constants from the `conversions` package here instead of "magic numbers"?

Reply:
👍 there is `conversions.PermissionAll` and `conversions.PermissionRead`